src/platform/packages/shared/kbn-esql-ast/src/commands/registry/join/utils.ts

```diff
@@ -272,7 +274,12 @@ export const createEnrichedGetByType = async (
       lookupSuggestions
     );
 
-    return [...markedSourceSuggestions, ...uniqueLookupSuggestions];
+    const categorizedLookupSuggestions = uniqueLookupSuggestions.map((suggestion) => ({
+      ...suggestion,
+      category: SuggestionCategory.LOOKUP_INDEX_FIELD,
```

Comment:
@bartoval remind me why we have to give a different category for this? Why is not falling on the typical field category? I would expect to only have a new category for the common ones and the rest to be SuggestionCategory.FIELD

Reply:
I added it only for clarity and to distinguish it, in case we want to change the rules in the future. The priority is the same as the FIELDs. I can remove it anyway.

Reply:
bc3d86e4fc2b90411450bf19fe7d70a462b3e2f5